_data/simple-icons.json

```diff
@@ -11764,6 +11764,12 @@
             "source": "https://github.com/wasmCloud/branding/",
             "guidelines": "https://github.com/wasmCloud/branding/"
         },
+        {
+            "title": "wasmer",
+            "hex": "4946DD",
+            "source": "https://github.com/wasmerio/wasmer.io/blob/0d425f5b4ace56496e75278e304f54492c46adde/public/images/icon.svg",
+            "guidelines": "https://wasmer.io/"
```

Comment:
I can't find the logo guidelines on this website. Unless I've overlooked it somewhere, please delete this field.